test/smoke/src/areas/positron/plots/plots.test.ts

```diff
@@ -53,6 +53,13 @@ test.describe('Plots', () => {
 				fail(`Image comparison failed with mismatch percentage: ${data.rawMisMatchPercentage}`);
 			}
 
+			if (!headless) {
+				await app.workbench.positronPlots.copyCurrentPlotToClipboard();
+
+				const clipboardImageBuffer = await app.workbench.positronClipboard.getClipboardImage();
+				expect(clipboardImageBuffer).not.toBeNull();
+			}
+
```

Comment:
🎉

Reply:
Wasn't able to get a headless run to test, but may as well include just in case.